src/zm_analysis_thread.h

```diff
@@ -1,10 +1,9 @@
 #ifndef ZM_ANALYSIS_THREAD_H
 #define ZM_ANALYSIS_THREAD_H
 
-#include "zm_thread.h"
-#include <signal.h>
-
 #include "zm_monitor.h"
+#include "zm_thread.h"
+#include <csignal>
```

Comment:
we don't actually use any of the signal stuff in analysis thread.  It can all go.

Reply:
Alirght, I removed the references to `<csignal>` from `AnalysisThread`.